From: =?utf-8?q?Guido=20G=C3=BCnther?= <agx@sigxcpu.org>
Date: Thu, 19 Feb 2009 16:18:13 +0100
Subject: [PATCH] gtk-vnc 0.3.8 handles rescaling with compiz

gtk-vnc 0.3.8 handle compositioning so scaling can be used in that case too.
---
 src/virtManager/details.py |    5 ++---
 1 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/src/virtManager/details.py b/src/virtManager/details.py
index 94a31c8..95e0696 100644
--- a/src/virtManager/details.py
+++ b/src/virtManager/details.py
@@ -222,9 +222,8 @@ class vmmDetails(gobject.GObject):
         else:
             self.vncViewer.set_keyboard_grab(False)
         self.vncViewer.set_pointer_grab(True)
-        if not topwin.is_composited():
-            self.vncViewer.set_scaling(True)
-            self.window.get_widget("details-menu-view-scale-display").set_active(True)
+        self.vncViewer.set_scaling(True)
+        self.window.get_widget("details-menu-view-scale-display").set_active(True)
 
         self.vncViewer.connect("vnc-pointer-grab", self.notify_grabbed)
         self.vncViewer.connect("vnc-pointer-ungrab", self.notify_ungrabbed)
-- 
